apps/admin/backend/src/app.ts

```diff
@@ -164,6 +218,19 @@ function buildApi({
       return null;
     },
 
+    getCurrentSystemSettings(): SystemSettings | null {
+      const currentSystemSettingsId = store.getCurrentSystemSettingsId();
```

Comment:
Do we plan to support multiple system settings objects in VxAdmin (like we plan to one day support multiple election objects)?

My hunch is that, unlike elections, we'll only ever store one system settings object at a time, since these settings are global / applicable to all elections. With that in mind, we could simplify and get rid of this "current system settings ID" persistence, knowing that there'll only ever be one system settings object. The only reason the analogous "current election ID" persistence exists is to one day support multiple elections.

Worth sanity checking that product assumption with Roe!

Reply:
I think in general I tend towards preserving optionality (eg. maybe we want to support a test mode settings profile and live mode one, where the test mode one has no poll worker card pins because a single official is simulating the role of multiple poll workers). But I'll check with Roe, and if we don't have a strong use case we should definitely simplify

Reply:
Removed support for multiple system settings and simplified here: https://github.com/votingworks/vxsuite/pull/3147/commits/fa25e9d6c2ad4f9e002ad5d9409a7fd5fdd5a09f